easybuild/easyconfigs/l/LIBSVM/LIBSVM-3.23_shared_lib.patch

```diff
@@ -0,0 +1,19 @@
+--- Makefile.orig	2019-11-04 10:10:12.475917486 +0100
```

Comment:
@tstrempel please add description and author to the top of the patch file?

Reply:
done